src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/common/LoginPageBase.java

```diff
@@ -1,11 +1,56 @@
 package com.qaprosoft.carina.demo.mobile.gui.pages.common;
 
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.demo.gui.components.FooterMenu;
 import org.openqa.selenium.WebDriver;
 
 import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.support.FindBy;
 
 public abstract class LoginPageBase extends AbstractPage {
 
+	@FindBy(id = "com.solvd.carinademoapplication:id/name")
```

Comment:
replace this id locator and all below on this locators (as it is native application):
@ExtendedFindBy(androidUIAutomator = "new UiSelector().resourceIdMatches(\".*id/name\")")